.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- Please see CONTRIBUTING.md for guidelines. -->
+
+## Environment
+
+* SuperCollider version:
+* Operating system:
+
+## Steps to reproduce
+
+<!-- Use triple backticks ``` ``` to delimit code. Make sure to use a minimal example. -->
+
+## Expected vs. actual behavior
+
+<!-- Paste error messages in entirety, delimited by ``` ```. Use gist if it's very long. -->
```

Comment:
why remove the crash report prompt? this is valuable information and IME most people do not provide it unless prompted directly

Reply:
restored!